process/lifecycle.md

```diff
@@ -9,101 +9,68 @@ has_children: true
 * Provide such additional information as the TAC may reasonably request.
-* Be available to present to the TAC with respect to the project’s proposal and inclusion in ASWF. Project teams should be prepared to present a detailed (20-30 minutes in length) overview on the project in addition to speaking to the information contained in the project contribution proposal.
-* Be deemed by the TAC to add potential value or value to the mission of ASWF.
+* Present the project’s proposal and inclusion in Academy Software Foundation to the TAC. Project teams should be prepared to present a detailed (20-30 minutes in length) overview of the project and speak to the information in the contribution proposal.
+* Be deemed by the TAC to add potential value or value to the mission of Academy Software Foundation.
 * Obtain an affirmative vote of the TAC.
 
 ### Benefits
 
 The Sandbox Stage benefits are outlined below.
 
-* Neutral hosting of the project’s community and any key assets (e.g. trademark, domain, etc.).
-* Access to the ASWF collaboration infrastructure ( including GitHub, JIRA, Confluence, mailing lists, Slack ).
+* Neutral hosting of the project’s community and key assets (e.g., trademark, domain, etc.).
+* Access to the Academy Software Foundation collaboration infrastructure ( including GitHub, JIRA, Confluence, mailing lists, and Slack ).
 * A sponsor from the TAC to assist the project in reaching the Incubation Stage and to facilitate collaboration with other project communities.
-* Right to refer to the project as a Sandbox Project of ASWF, and an opportunity to participate in events and other collaborative activities sponsored by ASWF.
-* Subject to applicable trademark usage guidelines, to display ASWF’s logo on the project’s code repository.
+* Right to refer to the project as a Sandbox Project of Academy Software Foundation and an opportunity to participate in events and other collaborative activities sponsored by Academy Software Foundation.
+* Subject to applicable trademark usage guidelines to display Academy Software Foundation’s logo on the project’s code repository.
+
+Sandbox Stage projects will not have an announcement blog or press announcement because projects at the Sandbox stage are in an early stage and still gaining alignment among the stakeholders. We often see projects in the Sandbox Stage change scope and focus; the TAC aims to ensure that projects have the space to focus on building a solid governance and contributor base to grow towards the Incubation Stage.
 
 ### Expectations
 
-Sandbox Stage projects should provide a quarterly report to the TAC outlining its progress on completing the requirements for the Incubation Stage.
+Sandbox Stage projects should provide a quarterly report to the TAC outlining its progress on completing the requirements for the Incubation Stage. If a project fails to provide these reports after a TAC request, the TAC may vote to remove the project from the Sandbox stage.
```

Comment:
I'm not sure that's a well known requirement, either on the part of the projects, or of the TAC?

Should this provide a scope for such a report, i.e. a paragraph, a Slack post, a 5 minute verbal update?